<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo">Sung-Huan Yu</a>
									<ul class="icons">
										<li><a href="https://twitter.com/silasysh" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
										<li><a href="https://www.facebook.com/silasyu" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
									</ul>
								</header>
							<!-- Banner -->
								<section id="banner">
									<div class="content">
										<header>
											<h2>Research</h2>
									  </header>
										    
									</div>
							</section>

							<!-- Content -->
								<section>
									<header class="main1">
										<h2>Genome annotation</h2>
									</header>
									<span class="image-wrapper">
										<img style="float: left; padding: 0px 100px 0px 0px;" src="images/genome_annotation.png" alt="" width="500" height="250"/>
                                                                        </span>
									<p>In order to interpret and translate omics dataset into biological and medical insights. 
                  Genome annotation is a basic requirement. I implemented ANNOgesic - the first tool for 
                  bacterial/archaeal RNA-Seq based genome annotations (S.H. Yu., 2018, GigaScience). ANNOgesic was reported and collected 
                  in many platforms and websites. Currently, I continue to extend ANNOgesic for more applications, 
                  and make it more user-friendly for the scientists who have no bioinformatic background.
                  Moreover, a annotation tool for eukaryotic species is also being developed. </p>
                  <br> <br>
                  <header class="main1">
										<h2>Multi-Omics data analysis</h2>
									</header>
									<span class="image-wrapper">
										<img style="float: left; padding: 0px 100px 0px 0px;" src="images/MultiOmics.png" alt="" width="500" height="350" />
                                                                        </span>
									<p>High-throughput technology revolutionized the whole field of biology. 
                  Numerous analyzing tools for NGS and proteomics data were published in the recent years. 
                  However, a single omics data may not reveal a complete picture for the global understanding. 
                  Every omics has different advantages, but contains some blind spots as well. For instance, 
                  transcriptomics can easily detect non-coding genes, but not a proper candidate for the studies of phonotype. 
                  Integration of multi-omics data definitely will be the discernible trend in the following years. 
                  Although the global correlation between transcriptomics and proteomics data is quite low (~10-25%) 
                  (Bai Y., 2015, Sci. Rep.; Xiong Q., 2015, Mol. Cel. Proteomics), many studies still successfully used 
                  mutli-omics to achieve specific goals like protein/RNA complex (Smirnov A., 2016, PNAS). 
                  Thus, the aims of my research in multi-omics analysis go for two directions â€“ improvement of the 
                  global integration for multi-omics data, and discovery of the issues which can be solved by multi-omics.</p>
                  <br>
                  <header class="main1">
										<h2>Single cell transcriptomic and proteomics</h2>
									</header>
									<p>Due to rapid progress of next-generation sequencing technologies, studying the characterization of 
                  individual cells can be achieved. In order to understand biological heterogeneity or study rare cells, 
                  single cell technology is the only way. In these recent years, the number of single cell studies in genome a
                  nd transcriptome level increases significantly, and the technology are reaching to maturity as well. 
                  However, single cell proteomics is still in an initial stage and numerous possible projects still can be explored. 
                  I am involved in some projects with the outstanding scientists of the single cell proteomics and transcriptomics community. 
                  The aim of the projects are developing a useful tool for single cell proteomics and transcriptomics.</p>
                  
                  <header class="main1">
										<h2>Deep learning/Machine learning</h2>
									</header>
									<span class="image-wrapper">
										<img style="float: left; padding: 0px 100px 0px 0px;" src="images/deep_learning.png" alt="" width="500" height="150"/>
                                                                        </span>
									<p>Deep learning is a subset of artificial intelligence which have catch a lot of attention recently. 
                  Currently, many studies show that deep learning performs high quality and accuracy predictions in bioinformatics field. 
                  However, applying deep learning technology to omics data analysis is still a fresh ground which needs to be reclaimed. 
                  Most of the tools for detecting the genomic features of NGS data are only based on reference sequence or reads. 
                  However, without the information from literatures or public databases, the details of the genomic features may not be 
                  able to be provided and the accuracy of identification may be influenced. Deep learning can build a model based on 
                  the publications and databases to improve the detection of genomic features.</p>
								  </section>             
						</div>
					</div>

                        <!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							<!-- Search -->
								<section id="search" class="alt">
									<form method="post" action="#">
										<input type="text" name="query" id="query" placeholder="Search" />
									</form>
								</section>

							<!-- Menu -->
								<nav id="menu">
									<header class="major">
										<h2>Menu</h2>
									</header>
									<ul>
										<li><a href="index.html">Homepage</a></li>
										<li><a href="about.html">About me</a></li>
										<li><a href="research.html">Research</a></li>
										<li><a href="software.html">Software</a></li>
										<li><a href="publication.html">Publications</a></li>
									</ul>
								</nav>
								<section>
									<header class="major">
										<h2>Get in touch</h2>
									</header>
									<p>welcome to contact me for the possibility of collaboration.</p>
									<ul class="contact">
										<li class="icon solid fa-envelope"><a href="#">shyu@biochem.mpg.de</a></li>
										<li class="icon solid fa-phone">+49 89 8578-2296</li>
										<li class="icon solid fa-home">Am Klopferspitz 18, 82152 Planegg<br />
										Germany</li>
									</ul>
								</section>

							<!-- Footer -->
								<footer id="footer">
									<p class="copyright">&copy; Untitled. All rights reserved. Demo Images: <a href="https://unsplash.com">Unsplash</a>. Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
								</footer>

						</div>
					</div>

                                </div>
		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
